numpy/doc/glossary.py

```diff
@@ -416,6 +433,12 @@
        Universal function.  A fast element-wise array operation.  Examples include
        ``add``, ``sin`` and ``logical_or``.
 
+   vectorized
```

Comment:
nit: `vectorization`, then use ``:term:`vectorized<vectorization>` `` above. Fine to leave to a follow-up

Reply:
adopted, but also expanded the content a bit.